code/datums/elements/movement_turf_changer.dm

```diff
@@ -26,7 +26,7 @@
 	SIGNAL_HANDLER
 
 	var/turf/destination = target.loc
-	if(!isturf(destination) || istype(destination, turf_type))
+	if(!isturf(destination) || istype(destination, turf_type) || istype(destination, /turf/open/openspace))
```

Comment:
unfortunately we have two types of openspace now, /turf/open/openspace and /turf/open/space/openspace so you should create a global typecache and use that

Reply:
Applied with https://github.com/tgstation/tgstation/pull/65757/commits/188915a62a48931c01928bebe478ec64466aec30